src/main/java/no/nav/pensjonbrevdata/mappers/sakBrev/SakBrevMap.java

```diff
@@ -42,61 +39,4 @@ public SakBrevMapper() {
         sakToBrevMap.put("KRIGSP", asList("PE_GP_01_010", "PE_IY_03_047", "PE_IY_03_048", "PE_IY_03_049", "PE_IY_03_051", "PE_IY_03_150", "PE_IY_03_151", "PE_IY_03_152", "PE_IY_03_153", "PE_IY_03_154",  "PE_IY_03_156", "PE_IY_03_157", "PE_IY_03_158", "PE_IY_03_159", "PE_IY_03_160", "PE_IY_03_161", "PE_IY_03_162", "PE_IY_03_163", "PE_IY_03_164", "PE_IY_03_168", "PE_IY_03_174", "PE_IY_03_175", "PE_IY_03_176", "PE_IY_03_177", "PE_IY_03_178", "PE_IY_03_179", "PE_IY_03_180", "PE_IY_04_050", "PE_IY_04_051", "PE_IY_04_060", "PE_IY_04_061", "PE_IY_05_006", "PE_IY_05_007", "PE_IY_05_008", "PE_IY_05_027", "PE_IY_05_041", "PE_IY_05_300", "PE_IY_05_301", "PE_IY_05_410", "PE_IY_05_411", "PE_IY_06_510", "PE_IY_06_511", "PE_OK_06_100", "PE_OK_06_101", "PE_OK_06_102", "PE_UT_04_002", "PE_IY_03_167", "AP_INFO_STID_MAN", "HENT_INFO_MAN"));
         sakToBrevMap.put("AFP_PRIVAT", asList("PE_AF_04_111", "PE_AF_04_112", "PE_AF_04_114", "PE_AP_04_216", "PE_GP_01_010", "PE_IY_03_047", "PE_IY_03_048", "PE_IY_03_049", "PE_IY_03_051", "PE_IY_03_150", "PE_IY_03_151", "PE_IY_03_152", "PE_IY_03_153", "PE_IY_03_154",  "PE_IY_03_156", "PE_IY_03_157", "PE_IY_03_158", "PE_IY_03_159", "PE_IY_03_160", "PE_IY_03_161", "PE_IY_03_162", "PE_IY_03_163", "PE_IY_03_164", "PE_IY_03_168", "PE_IY_03_174", "PE_IY_03_175", "PE_IY_03_176", "PE_IY_03_177", "PE_IY_03_178", "PE_IY_03_179", "PE_IY_03_180", "PE_IY_04_050", "PE_IY_04_051", "PE_IY_04_060", "PE_IY_04_061", "PE_IY_05_006", "PE_IY_05_007", "PE_IY_05_008", "PE_IY_05_027", "PE_IY_05_041", "PE_IY_05_300", "PE_IY_05_301", "PE_IY_05_410", "PE_IY_05_411", "PE_IY_06_103", "PE_IY_06_510", "PE_IY_06_511", "PE_OK_06_100", "PE_OK_06_101", "PE_OK_06_102", "PE_UT_04_002", "PE_IY_03_167", "VARSEL_REVURD", "AP_INFO_STID_MAN", "HENT_INFO_MAN", "VARSEL_TILBAKEBET",  "VEDTAK_TILBAKEKREV_MIDL", "AFP_INNV_MAN"));
     }
```

Comment:
Selv om disse endringene virker greie, kunne vi hatt en liten likhets-test? Risikoen er relativt lav, men konsekvensen kan være :skull: Vi skal forresten kun også fjerne redigerbare brev fra sakBrevMap senere. Pselv slår fortsatt opp i metadata for eksisterende journalposter for å finne ut om de skal vises til bruker eller ikke. Det blir slutt etter den gamle innboksen er helt skrudd av (har det skjedd enda?).